README.md

```diff
@@ -21,3 +14,51 @@ When we will start to support 2.9, the main TODO is to fully adapt the official
 
 * [usage](https://docs.ansible.com/ansible/2.9/user_guide/collections_using.html)
 * [development](https://docs.ansible.com/ansible/2.9/dev_guide/developing_collections.html)
+
+
+### Development setup
+
+* Install [poetry](https://python-poetry.org/docs/)
+* run `poetry install` anywhere inside the git checkout folder to install the current dependencies.
+* (`poetry update` can be used to re-generate the .lock file after making changes to dependencies)
+
+### Test this role using molecule
+
+Ensure you have hetzner cloud credentials:
+
+```
+env | grep HCLOUD_TOKEN > /dev/null || echo "[ERROR] Missing HCLOUD_TOKEN" && exit 1
+```
+
+At the moment, you also need AWS credentials in order to create DNS records. FUTUREWORK: remove dependency on AWS credentials.
```

Comment:
Can we even do this FUTUREWORK?

Reply:
Ah, by moving it to hetzer's DNS service, nvm.

Reply:
(yes, that was the idea)

Reply:
(it seem hetzner DNS doesn't support subzones - at least in the UI one cannot create something.wire.link, for instance.) Maybe let's stick with route53 for DNS then.